.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - '6'
+
+dist: trusty
+sudo: required
+
+env:
+  global:
+  - NODE_ENV=test
+  - LABS_AUTH_SERVICE_pgdb_password=''
+
+addons:
+  postgresql: "9.5"
```

Comment:
No need to resort to hack. I was wondering if travis offered the opportunity, but we can live with just 9.5